indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerConfig.java

```diff
@@ -242,9 +242,13 @@ public ShardSpec apply(HadoopyShardSpec input)
               )
           )
       );
+
+      Map<ShardSpec, HadoopyShardSpec> hadoopyShardSpecLookup = Maps.newHashMap();
```

Comment:
suggest rename to `innerHadoopShardSpecLookup` – "hadoopy" is easy to mistake for "hadoop" when someone is reading this code